### PR TITLE
 fix some typos

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: data/binding/convert_test.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func TestBoolToStringWithFormat(t *testing.T) {
 	assert.Nil(t, err)
 	v, err = s.Get()
 	assert.Nil(t, err)
-	assert.Equal(t, "truely", v)
```
Comment (andydotxyz) [inline data/binding/convert_test.go:47]: Sadly this line (and the one below) is not going to work, as the test is for "true"+"ly".
Feel free to make a slightly larger change that does not mark it as a typo, but correcting the spelling is what has caused the build failures.

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Thanks for looking at this. Unfortunately one of the changes breaks tests, as noted inline.

File: data/binding/convert_test.go (hunk 1 above)
Comment (cuishuang) [reply data/binding/convert_test.go]: Thanks.

**Review by andydotxyz — APPROVED** [review]
Thanks.

If you start working on larger changes please work on `develop` or the latest `release` branch as we don't normally work on `master`.

**Pull request merged**